samples/sampler/stories/default/ProgressBar.js

```diff
@@ -1,49 +1,62 @@
-// 			const position = select('position', ['', 'above', 'above left', 'above right', 'above before', 'above after', 'before', 'left', 'right', 'after', 'below', 'below left', 'below right', 'below before', 'below after'], ProgressBarTooltipConfig, '');
-// 			const side = nullify(select('side (Deprecated)', ['', 'after', 'before', 'left', 'right'], ProgressBarTooltipConfig, ''));
-//
-// 			return (
-// 				<ProgressBar
-// 					backgroundProgress={number('backgroundProgress', ProgressBarConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.5)}
-// 					disabled={disabled}
-// 					highlighted={boolean('highlighted', ProgressBarConfig)}
-// 					orientation={select('orientation', ['horizontal', 'vertical'], ProgressBarConfig, 'horizontal')}
-// 					progress={number('progress', ProgressBarConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.4)}
-// 					style={{marginLeft: ri.scaleToRem(72), marginRight: ri.scaleToRem(72)}}
-// 				>
-// 					{tooltip ? (
-// 						<ProgressBarTooltip
-// 							position={position}
-// 							side={side}
-// 						/>
-// 					) : null}
-// 				</ProgressBar>
-// 			);
-// 		},
-// 		{
-// 			info: {
-// 				text: 'The basic ProgressBar'
-// 			}
-// 		}
-// 	);
+import ProgressBar, {ProgressBarTooltip} from '@enact/moonstone/ProgressBar';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, number, select} from '@enact/storybook-utils/addons/controls';
+import ri from '@enact/ui/resolution';
+
+ProgressBar.displayName = 'ProgressBar';
+const ProgressBarConfig = mergeComponentMetadata(
+	'ProgressBar',
+	ProgressBar
+);
+
+ProgressBarTooltip.displayName = 'ProgressBarTooltip';
+const ProgressBarTooltipConfig = mergeComponentMetadata(
+	'ProgressBarTooltip',
+	ProgressBarTooltip
+);
+
+export default {
+	title: 'Moonstone/ProgressBar',
+	component: 'ProgressBar'
+};
+
+export const _ProgressBar = (args) => {
+	// added here to force Storybook to put the ProgressBar tab first
+	const disabled = args['disabled'];
+
+	// tooltip is first so it appears at the top of the tab. the rest are alphabetical
+	const tooltip = args['tooltip'];
+	const position = args['position'];
+
+	return (
+		<ProgressBar
+			backgroundProgress={args['backgroundProgress']}
+			disabled={disabled}
+			highlighted={args['highlighted']}
+			orientation={args['orientation']}
+			progress={args['progress']}
+			style={{marginLeft: ri.scaleToRem(72), marginRight: ri.scaleToRem(72)}}
+		>
+			{tooltip ? (
+				<ProgressBarTooltip
+					position={position}
+				/>
+			) : null}
+		</ProgressBar>
+	);
+};
+
+boolean('disabled', _ProgressBar, ProgressBarConfig);
+boolean('highlighted', _ProgressBar, ProgressBarConfig);
+boolean('tooltip', _ProgressBar, ProgressBarTooltipConfig);
+number('backgroundProgress', _ProgressBar, ProgressBarConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.5);
+number('progress', _ProgressBar, ProgressBarConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.4);
```

Comment:
Change the progress and backgroundProgress control from number to range. It seems that with number is not working properly